src/helper/common.ts

```diff
@@ -1,28 +1,41 @@
-import { Identity } from "many-js";
-import { Account } from "../store/accounts";
+import { Identity, ANON_IDENTITY } from "many-js"
+import { Account } from "../store/accounts"
 
 export const parseIdentity = (key: any): string => {
   if (key === undefined) {
-    return "<oaa>";
+    return ANON_IDENTITY
   }
-  const identity = Identity.fromPublicKey(key);
+  const identity = Identity.fromPublicKey(key)
 
-  return `<${identity.toString()}>`;
-};
+  return `${identity.toString()}`
+}
 
 export const getAddressFromHex = (hex: any): string => {
-  const identity = Identity.fromHex(hex);
-  return identity.toString();
-};
+  const identity = Identity.fromHex(hex)
+  return identity.toString()
+}
 
 export const makeShortId = (idString: string): string =>
-  `<${idString.slice(0, 4)}...${idString.slice(-4)}>`
+  `${idString.slice(0, 4)}...${idString.slice(-4)}`
 
 export function displayId(account: Account): { full: string; short: string } {
   if (!account?.keys) {
-    return { full: "", short: `<oaa>` }
+    return { full: "", short: ANON_IDENTITY }
   }
   const identity = Identity.fromPublicKey(account.keys.publicKey)
   const idString = identity.toString()
   return { full: idString, short: makeShortId(idString) }
-}
\ No newline at end of file
+}
+
+export const parseNumberToBigInt = (v: number) =>
+  BigInt(Math.round(v * 10 ** 9))
```

Comment:
You should use a constant for this as you seem to be using it in many places.

Also, just a note; 9 decimal digits is only for FBT. We will have that information provided by the ledger later on as we add more information regarding tokens, but it will vary from token to token.

Reply:
> You should use a constant for this as you seem to be using it in many places.
> 
> Also, just a note; 9 decimal digits is only for FBT. We will have that information provided by the ledger later on as we add more information regarding tokens, but it will vary from token to token.

the decimal digits will now come from the args and defaulted to 9